sys/include/net/conn/udp.h

```diff
@@ -93,6 +96,8 @@ int conn_udp_getlocaladdr(conn_udp_t *conn, void *addr, uint16_t *port);
  *
  * @note    Function may block.
  *
+ * @note    This function needs to be called from the same function as @ref conn_udp_create.
```

Comment:
Ah, now I saw the problem. You were to the second instance of the word `function`. That was simply a typo.